app/components/list-item/index.tsx

```diff
@@ -1,26 +1,54 @@
+'use client'
+
 import { IListItemType } from '@/app/app-interfaces'
 import { appLabels } from '@/app/constants'
+import Image from 'next/image'
+import editIcon from '../../../images/pencil-icon.png'
+import deleteIcon from '../../../images/trash-icon.png'
+import { atom, useAtom } from 'jotai'
 
-interface IListItemProps {
-  items: IListItemType[]
-}
+export const todoListAtom = atom<IListItemType[]>([])
+
+const ListItem = () => {
+  const [todoList, setTodoList] = useAtom(todoListAtom)
+
+  const handleTodoStatusChange = (itemId: string) => {
+    const updatedTodos = todoList.map((todo) => {
+      if (todo.id === itemId) {
+        if (todo.status === 'Done') todo.status = 'Active'
```

Comment:
Try to prefer ternary operator for single if else

Reply:
Updated the nested if else by using ternary operator.